packages/arb-util/configuration/configuration.go

```diff
@@ -200,9 +205,13 @@ func Parse(ctx context.Context) (*Config, *Wallet, *ethutils.RPCEthClient, *big.
 		return nil, nil, nil, nil, errors.Wrap(err, "error getting --l1.url")
 	}
 
+	if len(l1URL) == 0 {
+		return nil, nil, nil, nil, errors.New("required parameter --l1.url is missing")
+	}
+
```

Comment:
Do we want to support a loading everything including this from a config file?

Reply:
yup, loading everything from config file is supported